src/main/java/touro/chess/ChessController.java

```diff
@@ -0,0 +1,18 @@
+package touro.chess;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+
+import java.util.ArrayList;
+
+public class ChessController {
+
+    @FXML
+    ArrayList<Label> lightLabels, darkLabels;
```

Comment:
Why would we need the dark labels and light labels?

Reply:
Only needed it for style. Now they are all in one ArrayList.